js/app.js

```diff
@@ -1,234 +1,139 @@
-
-//Time variable
-var time =['6am-7am', '7am-8am', '8am-9am', '9am-10am', '10am-11am', '11am-12pm', '12pm-1pm', '1pm-2pm', '2pm-3pm', '3pm-4pm', '4pm-5pm', '5pm-6pm', '6pm-7pm','7pm-8pm',];
-
-//The Cities
-var pike = {
-  minP : 23,
-  maxP : 65,
-  avgP: 6.3,
-  pikeCustomers: function(){
-    // var min = Math.round(pike.minP);
-    // var max = Math.round(pike.maxP);
-    return Math.round(Math.random() * (pike.maxP - pike.minP + 1)) + pike.minP;
-
-  },
-  pikeCookies: function(){
-    return Math.round(pike.pikeCustomers() * pike.avgP);
-  },
-  pikeTime: function(){
-    var firstAndPike = document.getElementById('1stAndPike');
-    var total = 0;
-    for(var i = 0; i < time.length; i++){
-      var pikeEl = document.createElement('li');
-      var randomNum =pike.pikeCookies();
-      pikeEl.textContent = '1st and Pike sold ' + randomNum + ' cookies from '+ time[i] + '.';
-      total += randomNum;
-      firstAndPike.appendChild(pikeEl);
-    }
-    var totalCookies = document.createElement('li');
-    totalCookies.textContent = 'Total number of cookies sold at 1st and Pike is ' + total;
-    firstAndPike.appendChild(totalCookies);
+
+var timeTable =['6am-7am', '7am-8am', '8am-9am', '9am-10am', '10am-11am', '11am-12pm', '12pm-1pm', '1pm-2pm', '2pm-3pm', '3pm-4pm', '4pm-5pm', '5pm-6pm', '6pm-7pm','7pm-8pm',];
+var allCities=[];
+var totalAmountForAllCities = [];
+var totalAmountEachHour=[];
+
+var cookieBusiness = document.getElementById('cookieData');
+
+
+function City(storeName,minP,maxP,avgP) {
+  this.storeName=storeName;
+  this.minP = minP;
+  this.maxP = maxP;
+  this.avgP = avgP;
+  this.customersPerHour = [];
+  this.customers();
+  this.cookiesPerHour = [];
+  this.cookies();
+  this.totalCookies = [];
+  this.total();
+  allCities.push(this);
+}
+
+
+City.prototype.customers = function(){
+  for (var i =0; i < timeTable.length; i++ ){
+    this.customersPerHour.push(Math.round(Math.random() * (this.maxP - this.minP)) + this.minP);
   }
 };
 
-var seaTac= {
-  minP: 3,
-  maxP: 24,
-  avgP: 1.2,
-  seaTacCustomers: function(){
-    return Math.round(Math.random() * (seaTac.maxP - seaTac.minP+1)) + seaTac.minP;
-  },
-  seaTacCookies: function(){
-    return Math.round(seaTac.seaTacCustomers() * seaTac.avgP);
-
-  },
-  seaTacTime: function(){
-    var seattleTacAiport = document.getElementById('SeaTacAirport');
-    var total = 0;
-    for(var i = 0; i < time.length; i++){
-      var seaTacEl = document.createElement('li');
-      var randomNum =seaTac.seaTacCookies();
-      seaTacEl.textContent = 'The SeaTac Airport sold ' + randomNum + ' cookies from '+ time[i] + '.';
-      total += randomNum;
-      seattleTacAiport.appendChild(seaTacEl);
-    }
-    var totalCookies = document.createElement('li');
-    totalCookies.textContent = 'Total number of cookies sold at the Sea Tac airport is ' + total;
-    seattleTacAiport.appendChild(totalCookies);
-  },
+
```

Comment:
Looks like you are using some redundant functions to produce your random sales quantities. Try pulling them into their own function that can then be called where needed. Remember DRY principles.